src/client/datascience/jupyter/jupyterExecution.ts

```diff
@@ -144,18 +143,10 @@ export class JupyterExecutionBase implements IJupyterExecution {
                     // Create a server tha  t we will then attempt to connect to.
                     result = this.serviceContainer.get<INotebookServer>(INotebookServer);
 
-                    // Populate the launch info that we are starting our server with
-                    const launchInfo: INotebookServerLaunchInfo = {
-                        connectionInfo: connection!,
-                        workingDir: options ? options.workingDir : undefined,
-                        uri: options ? options.uri : undefined,
-                        purpose: options ? options.purpose : uuid(),
-                        disableUI: !allowUI
```

Comment:
None of the properties of `INotebookServerLaunchInfo ` were used, only `connectionInfo` was used everywhere.
Hence removed `INotebookServerLaunchInfo`